DemoApp/Sources/Views/Reactions/ReactionOverlayView.swift

```diff
@@ -25,20 +27,19 @@ struct ReactionOverlayView: View {
     @ViewBuilder
     private func reactionView(for reaction: Reaction) -> some View {
         ZStack {
-            BottomRightView {
-                ReactionIcon(iconName: reaction.iconName).padding()
+            TopRightView {
```

Comment:
Yeap, reactions are now positioned on the top right on all clients. The reactions aren't visible because you need to move the floating view, right? Ideally we should have some logic in place that overlay elements (options, info, connection and reactions) are reacting to the position of the floating view but it would be better to get it done in another PR.

Reply:
Yes, not in the scope of this PR, but please raise it with UX. The default experience is kind of broken from a design perspective.